### ISSUE TITLE
pipelinerun describe nil pointer

### ISSUE THREAD

**Issue by Conky5** [tektoncd/cli#202]
# Expected Behavior

`tkn pipelinerun describe <pipeline-run>` should always show the status of an existing pipelinerun without error.

# Actual Behavior

`tkn pipelinrun describe <pipeline-run>` can throw the following error immediately after a pipelinerun is created:
```
Failed to execute templateError: template: Describe Pipelinerun:10:25: executing "Describe Pipelinerun" at <.PipelineRun.Status.StartTime>: dereference of nil pointer of type v1.Time
```

# Steps to Reproduce the Problem

1. Generate a `pipelinerun`
2. Immediately run a `describe` on it

# Additional Info

**Comment by vdemeester** [comment]
@Conky5 which version are you running ?
/kind bug
/priority critical-urgent

**Comment by Conky5** [comment]
@vdemeester we're seeing this on `v0.2.0` and latest `master` 8d221a41b926b31c8809fb6bdb5f2d4d1c0266f4.

**Comment by vdemeester** [comment]
@Conky5 thx for the report :angel: We will fix that and do a 0.2.1 release :angel:

**Comment by hrishin** [comment]
This is fixed now. Thank you @Conky5 for reporting this issue. :+1:
